### DISCUSSION TITLE
Ability to edit and change Master Slide?

### DISCUSSION THREAD

**Discussion by sanjevirau** [ShapeCrawler/ShapeCrawler#195] [Ideas]
Hi there,
Thank you so much for this library. I'm having a use case where:

There are multiple Master Slides in my PPTX and I want to move all my slides into only One Master Slide and remove the other Master Slides.
I want to change the Theme Color of my PPTX programmatically. Since the first use-case makes all the Master Slide into one, thus only need to change the theme color for the one Master Slide it has.
I hope the use case is understandable. (If not, do let me know). Can ShapeCrawler help me achieve this?

And if not, can I know if this use-case is achievable programmatically or not.

Thank you :)

**Comment by AdamShakhabov** [comment]
Hello @sanjevirau🙂 Sorry, I only recently noticed, that your answer is without reply.

Unfortunately, the library still doesn't have the corresponding APIs to achieve the above scenario. 

I've created a couple of issues, whose implementations should make it possible:

- https://github.com/ShapeCrawler/ShapeCrawler/issues/368
- https://github.com/ShapeCrawler/ShapeCrawler/issues/369
- https://github.com/ShapeCrawler/ShapeCrawler/issues/370
